config/betterquesting/Readme.md

```diff
@@ -1,106 +1,114 @@
-607. Send in the pull request.  The team will assign reviewers.
-608. If they have comments, you can add new commits to your branch and push them to github, and they will automatically show up in the PR. You can keep doing more commits while the PR is open.
-609. Once the PR is accepted, you can delete your branch. This can also be done when the PR is accepted from a local branch.
-610. When doing new work after the PR is accepted, make sure you checkout master and do a pull to update it. Creating a new branch from the latest master is easiest. You can reuse the branch name you had previously if you switch to it, and right-click on the master commit and choose merge. 
-611. To update your forked repo, you have to setup git to point to two repos. It might be easier to just delete your forked repo and fork again if you need to update. Another option is to do a Pull Request from the source repo into your fork repo.
-
-# Translating the quest book #
-701. Whenever the quest book is saved with `/bq_admin default save`, the saved quest book data will contain `en_US.lang`, which is a template for translating the quest book. By itself, this file does nothing, since it just contains the default English text for the quests.
-702. NOTE: `en_US.lang` contains all quest and quest line text, and will likely be changed by any quest book edits. However, if you do get a merge conflict on this file, dealing with it is actually pretty easy: it gets regenerated every time the quest book is saved, so you can just ignore the conflicts in this file and re-save the quest book, and commit the regenerated copy.
-703. Translators can work off of the translation keys in `en_US.lang` to create a translated lang file.
-704. It is also possible to perform the translation directly in the quest book in-game. Use `/bq_admin default save translated` to export the changes to `saved_quests/translated/en_US.lang`, and then extract the translated entries manually. The `diff` command would probably be helpful here, diffing against the original `en_US.lang`.
-705. Just drop the translated lang file into `config/txloader/load/betterquesting/lang` to apply the translations. In previous versions of BetterQuesting, it was necessary to load a custom version of the quest book data for translations to work, but this is no longer needed.
+
+1. Open the quest book and click Edit at the bottom.
+2. Click the tab you want to put the quest, then click Designer.
+3. Click New Quest, and place it in the quest graph.
+4. Click Select Icon and choose an appropriate icon for the quest.
+5. To define requirements, select Edit, then Requirements, then search for the requirement quest on the right hand side with the quest ID or the title. Select + to add the quest to the requirements. The eye symbol lets you select if the connection is always shown/always hidden/only shown with hovering over or shift. Ideally you want to show dependencies but avoid always shown if it leads to excessive line crossing.
```

Comment:
Is it worth mentioning that you can right-click quests to copy the quest ID? Not sure how easy that feature is to discover